src/components/Activity/ActivityHub.js

```diff
@@ -48,87 +72,109 @@ ActivityHub.propTypes = {
   activities: PropTypes.arrayOf(PropTypes.object).isRequired,
   keys: PropTypes.func.isRequired,
   clearActivity: PropTypes.func.isRequired,
+  getAppByProxyAddress: PropTypes.func.isRequired,
 }
 
-function ActivityItem({ ready, activity }) {
-  const [showPayload, setShowPayload] = React.useState(true)
+function ActivityItem({ activity, getAppByProxyAddress, ready }) {
+  // const [showPayload, setShowPayload] = React.useState(true)
```

Comment:
In case of doubt, this state can be removed. I think they're remnants of the original PR.